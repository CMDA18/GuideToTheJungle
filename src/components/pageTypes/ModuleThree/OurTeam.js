// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../../Grid'
import PageTitle from '../../PageTitle'
import Intro from '../../Intro'
import { ourTeam } from '../../../constants/content/index'

type Props = {
  test: string
}

class OurTeam extends Component<Props> {
  render () {
    return (
      <div className='ourTeam' data-test={this.props.test}>
        <Helmet
          title='ourTeam'
        />
        <Grid page='ourTeam'>
          <header>
            <PageTitle {...ourTeam}/>
            <Intro {...ourTeam}/>
          </header>
        </Grid>
      </div>
    )
  };
}

export default OurTeam
