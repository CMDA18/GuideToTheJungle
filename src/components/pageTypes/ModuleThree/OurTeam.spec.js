import React from 'react'
import { shallow } from 'enzyme'
import OurTeam from './OurTeam'

it('renders without crashing', () => {
  expect(shallow(<OurTeam />)).toMatchSnapshot()
})
