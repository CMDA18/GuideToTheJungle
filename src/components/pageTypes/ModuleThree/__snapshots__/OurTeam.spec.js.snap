// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OurTeam />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="ourTeam"
        />,
        <Grid
          page="ourTeam"
        >
          <header>
            <PageTitle
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="3.1 onze team"
            />
            <Intro
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="3.1 onze team"
            />
          </header>
        </Grid>,
      ],
      "className": "ourTeam",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "ourTeam",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <header>
            <PageTitle
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="3.1 onze team"
            />
            <Intro
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="3.1 onze team"
            />
          </header>,
          "page": "ourTeam",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <PageTitle
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="3.1 onze team"
              />,
              <Intro
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="3.1 onze team"
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                "introTitle": "Onze evolutie in tijd",
                "pageTitle": "3.1 onze team",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                "introTitle": "Onze evolutie in tijd",
                "pageTitle": "3.1 onze team",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "header",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="ourTeam"
          />,
          <Grid
            page="ourTeam"
          >
            <header>
              <PageTitle
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="3.1 onze team"
              />
              <Intro
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="3.1 onze team"
              />
            </header>
          </Grid>,
        ],
        "className": "ourTeam",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "ourTeam",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <header>
              <PageTitle
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="3.1 onze team"
              />
              <Intro
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="3.1 onze team"
              />
            </header>,
            "page": "ourTeam",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <PageTitle
                  introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                  introTitle="Onze evolutie in tijd"
                  pageTitle="3.1 onze team"
                />,
                <Intro
                  introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                  introTitle="Onze evolutie in tijd"
                  pageTitle="3.1 onze team"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                  "introTitle": "Onze evolutie in tijd",
                  "pageTitle": "3.1 onze team",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                  "introTitle": "Onze evolutie in tijd",
                  "pageTitle": "3.1 onze team",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "header",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
